app/models/data_resources/event_record.rb

```diff
@@ -58,7 +61,16 @@ class EventRecord < ApplicationRecord
     events_in_timespan = joins(:dates).where(fixed_dates: { id: list_of_fixed_dates.map(&:id) })
     events_in_timespan.map do |event_record|
       # return the start_date of the event if the requested start_date is before the event start_date
-      event_record.in_date_range_start_date = start_date < event_record.dates.first.date_start ? event_record.dates.first.date_start : start_date
+      if start_date < event_record.dates.first.date_start
+        event_record.in_date_range_start_date = event_record.dates.first.date_start
+      else
+        if event_record.recurring?
+          # ignore the first date, which is the original date, and use the second (closest upcoming)
+          event_record.in_date_range_start_date = event_record.dates.second.date_start
+        else
+          event_record.in_date_range_start_date = start_date
+        end
+      end
```

Comment:
this was necessary for our very own `list_date` logic to respect recurring events (803de6c3dd9f02f18d62500bed0349018768864d)